iso15118/shared/messages/datatypes.py

```diff
@@ -473,8 +474,11 @@ class PVEVSEPresentVoltageDin(PVEVSEPresentVoltage):
     See section 9.5.2.4 in DIN SPEC 70121
 
     In DIN the Element unit is optional, in ISO it is mandatory.
+    In DIN there is no range for the value specified.
+    There are EVs that sometimes send values below zero.
```

Comment:
this is for the PVEVSE, so I guess you meant here EVSE and not EV?

Reply:
Indeed it is at the wrong place here. It is meant for the Datatype PVEVTargetCurrentDin. Because Skoda Enyaq has sent a value of -5 A in the beginning of the charging session.

I will fix it next week. Because of holidays :)